pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -529,7 +529,7 @@ protected void txnAckTest(boolean batchEnable, int maxBatchSize,
             // after transaction abort, the messages could be received
             Transaction commitTxn = getTxn();
             for (int i = 0; i < messageCnt; i++) {
-                message = consumer.receive(2, TimeUnit.SECONDS);
+                message = consumer.receive();
```

Comment:
If the network is not good, here will fail. so no need to add timeout for this.
https://github.com/apache/pulsar/actions/runs/5899745435/job/16003227755?pr=21008